Delcam.ProductInterface.PowerSHAPE.Test/DatabaseEntity Tests/Entity Tests/MeshTest.cs

```diff
@@ -249,6 +250,52 @@ public void MinimumDistanceMeshCompCurve()
             //Assert.AreEqual(0.0, entity1.DistanceTo(entity2));
         }
 
+        /// <summary>
+        /// When writing mesh to a DMT check that mesh is kept the same: the same bounding box,
+        /// the same number of triangles
+        /// </summary>
+        [Test]
+        public void WhenWriteToDMTFile_CheckMeshDidNotChange()
+        {
+            // Import mesh
+            _powerSHAPE.Reset();
+            var importedMesh =_powerSHAPE.ActiveModel.Meshes.CreateMeshFromFile(new File(TestFiles.SINGLE_MESH));
+            var numberOfTriangles = importedMesh.NumberOfTriangles;
+            var boundingBox = importedMesh.BoundingBox;
+
+            // Export mesh 
+            var output = File.CreateTemporaryFile("dmt");
```

Comment:
You need to clean up this temporary file.  Preferably do it before any Asserts so that the file is cleaned up even if the test fails.

Reply:
Done